tests/track/test_track.py

```diff
@@ -0,0 +1,187 @@
+YieldFixture = Generator[T, None, None]
+
+
+@pytest.fixture
+def test_track_dir(test_data_dir: Path) -> Path:
+    return test_data_dir / "track"
+
+
+@pytest.fixture
+def test_track_tmp_dir(
+    test_data_tmp_dir: Path, test_track_dir: Path
+) -> YieldFixture[Path]:
+    # Create tmp dir
+    test_track_tmp_dir = test_data_tmp_dir / "track"
+    test_track_tmp_dir.mkdir(exist_ok=True)
+    # Copy test files to tmp dir
+    shutil.copytree(test_track_dir, test_track_tmp_dir, dirs_exist_ok=True)
+    # Delete tracks files from tmp dir (we create them during the tests)
+    extension = CONFIG["DEFAULT_FILETYPE"]["TRACK"]
+    tracks_files_to_delete = test_track_tmp_dir.rglob(f"*{extension}")
+    for f in tracks_files_to_delete:
+        f.unlink()
+    # Yield tmp dir
+    yield test_track_tmp_dir
+    # Teardown tmp dir after use in test
+    shutil.rmtree(test_track_tmp_dir)
+
+
+@pytest.mark.parametrize(
+    "test_case, sigma_l, sigma_h, sigma_iou, t_min, t_miss_max",
+    [
+        ("default", SIGMA_L, SIGMA_H, SIGMA_IOU, T_MIN, T_MISS_MAX),
+        ("sigma_l_0_1", 0.1, SIGMA_H, SIGMA_IOU, T_MIN, T_MISS_MAX),
+        ("sigma_l_0_5", 0.5, SIGMA_H, SIGMA_IOU, T_MIN, T_MISS_MAX),
+        ("sigma_h_0_2", SIGMA_L, 0.2, SIGMA_IOU, T_MIN, T_MISS_MAX),
+        ("sigma_h_0_6", SIGMA_L, 0.6, SIGMA_IOU, T_MIN, T_MISS_MAX),
+        ("sigma_iou_0_2", SIGMA_L, SIGMA_H, 0.2, T_MIN, T_MISS_MAX),
+        ("sigma_iou_0_6", SIGMA_L, SIGMA_H, 0.6, T_MIN, T_MISS_MAX),
+        ("t_min_3", SIGMA_L, SIGMA_H, SIGMA_IOU, 3, T_MISS_MAX),
+        ("t_min_10", SIGMA_L, SIGMA_H, SIGMA_IOU, 10, T_MISS_MAX),
+        ("t_miss_max_25", SIGMA_L, SIGMA_H, SIGMA_IOU, T_MIN, 25),
+        ("t_miss_max_75", SIGMA_L, SIGMA_H, SIGMA_IOU, T_MIN, 75),
+    ],
+)
+def test_track_pass(
+    test_track_dir: Path,
+    test_track_tmp_dir: Path,
+    test_case: str,
+    sigma_l: float,
+    sigma_h: float,
+    sigma_iou: float,
+    t_min: int,
+    t_miss_max: int,
+) -> None:
+    """Tests the main function of OTVision/track/track.py
+    tracking detections from .otdet files (from short test videos)
+    using the IOU tracker by Bochinski et al.
+    """
+
+    # Track all test detections files
+    track(
+        paths=[test_track_tmp_dir / test_case],
+        sigma_l=sigma_l,
+        sigma_h=sigma_h,
+        sigma_iou=sigma_iou,
+        t_min=t_min,
+        t_miss_max=t_miss_max,
+    )
+
+    # Get reference tracks file names
+    extension = CONFIG["DEFAULT_FILETYPE"]["TRACK"]
+    ref_tracks_files = (test_track_dir / test_case).glob(f"*{extension}")
+    tracks_file_names = [file.name for file in ref_tracks_files]
+
+    # Compare all test tracks files to their respective reference tracks files
+    equal_files, different_files, irregular_files = cmpfiles(
+        a=test_track_dir / test_case,
+        b=test_track_tmp_dir / test_case,
+        common=tracks_file_names,
+        shallow=False,
+    )
+    assert tracks_file_names == equal_files
```

Comment:
Can we check if files in `equal_files` are contained in  `tracks_file_names`, thus ignoring ordering just to be on the safe side in case cmpfiles returns a list with elements in different order?
```suggestion
    for equal_file in equal_files:
        assert equal_file in tracks_file_names
```